db/schema.rb

```diff
@@ -16,21 +16,22 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
   enable_extension "hstore"
+  enable_extension "pg_stat_statements"
 
   create_table "active_admin_comments", force: :cascade do |t|
-    t.string   "resource_id",   null: false
-    t.string   "resource_type", null: false
+    t.string   "resource_id",   limit: 255, null: false
```

Comment:
It seems like we you and me have different configuations... mine seems to want to remove limits and yours is adding them.. I wonder what the difference is?